db/warehouse/migrate/20250304143108_add_project_id_to_hud_clients.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AddProjectIdToHudClients < ActiveRecord::Migration[7.0]
+  def change
+    ['hud_report_dq_clients', 'hud_report_path_clients'].each do |table|
```

Comment:
Do we need a data migration as well, or does the view handle a missing `project_id` appropriately?